react-common/styles/onboarding/TeachingBubble.less

```diff
@@ -32,6 +32,16 @@
     border-radius: 0.5rem;
     padding: 1rem;
     z-index: @modalDimmerZIndex;
+
+    .common-button {
+
+        &:focus {
```

Comment:
can we make this darken the button a bit as well? It's pretty subtle with just a small extra outline, especially on the white button

Reply:
@jwunderl @thsparks our buttons generally only darken on hover, do you want me to add this effect for on focus for all buttons or just for the teaching bubble?

Reply:
I did it for the teaching bubble only, but using the same darkening method we use for on hover:
![tabButton2](https://user-images.githubusercontent.com/15070078/229847813-5076fc47-39b2-4388-ba02-0a0c08c62dce.gif)

Reply:
I'm going to merge, but feel free to file an issue if I should make any other change.